fluent-langneg/src/negotiate_languages.ts

```diff
@@ -51,22 +51,19 @@ export interface NegotiateLanguagesOptions {
 export function negotiateLanguages(
   requestedLocales: Readonly<Array<string>>,
   availableLocales: Readonly<Array<string>>,
-  {
-    strategy = "filtering",
-    defaultLocale,
-  }: NegotiateLanguagesOptions = {}
+  { strategy = "filtering", defaultLocale }: NegotiateLanguagesOptions = {}
 ): Array<string> {
-
   const supportedLocales = filterMatches(
-    Array.from(Object(requestedLocales)).map(String),
-    Array.from(Object(availableLocales)).map(String),
+    Array.from(requestedLocales ?? []).map(String),
+    Array.from(availableLocales ?? []).map(String),
```

Comment:
This has the same effect of avoiding throws for nullish inputs as before, but is clearer & type-friendlier about it.